docs-markdown/src/controllers/master-redirect-controller.ts

```diff
@@ -193,9 +193,15 @@ export function generateMasterRedirectionFile() {
                         if (fs.existsSync(docsRedirectDirectory)) {
                             fs.mkdirSync(deletedRedirectsPath);
                         } else {
-                            fs.mkdirSync(docsAuthoringHomeDirectory);
-                            fs.mkdirSync(docsRedirectDirectory);
-                            fs.mkdirSync(deletedRedirectsPath);
+                            if (!fs.existsSync(docsAuthoringHomeDirectory)) {
```

Comment:
@jamarw this was getting stuck for me because I already had the directory from the templates. You might want to check that this doesn't get hung up if the directory already exists when trying to use the templates.